static/css/main.css

```diff
@@ -223,424 +225,422 @@ body {
-    line-height: 13px;
-  }
+    .bgStatus .currentBG.bg-limit, .bgStatus .currentBG.icon-hourglass {
+      font-size: 50px;
+    }
 
-  #silenceBtn a {
-    font-size: 20px;
-  }
+    .bgStatus .currentDirection {
+        font-size: 50px;
+        line-height: 50px;
+    }
 
-  .status {
-    padding-top: 0;
-    font-size: 20px;
-  }
+    .bgStatus .currentDetails {
+        font-size: 15px;
+    }
 
-  .statusBox {
-    width: auto;
-  }
+    .time {
+        font-size: 50px;
+        line-height: 40px;
+        width: 250px;
+    }
 
-  #currentTime {
-    display: inline;
-    vertical-align: middle;
-  }
+    #chartContainer {
+        top: 165px;
+    }
+    #chartContainer svg {
+        height: calc(100vh - 165px);
+    }
 
-  .statusPills {
-    display: inline;
-    margin-left: auto;
-    font-size: 12px;
-    line-height: 13px;
-  }
-
-  .focus-range {
-    position: absolute;
-    top: 60px;
-    left: auto;
-    right: 10px;
-    margin: 0;
-    width: auto;
-  }
+}
 
-  .focus-range li {
-    display: block;
-  }
+@media (max-width: 400px) {
+    .status {
+        text-align: center;
+        margin-bottom: 0;
+        height: 152px;
+    }
+
+    .bgStatus {
+        float: none;
+        padding: 0;
+        text-align: center;
+        width: 250px;
+    }
+
+
+    .bgButton {
+        margin: 5px;
+        width: auto;
+    }
+
+    .bgStatus .currentBG {
+        font-size: 70px;
+        line-height: 70px;
+    }
+
+    .bgStatus .currentBG.bg-limit, .bgStatus .currentBG.icon-hourglass {
+        font-size: 60px;
+        padding-left: 20px;
+    }
+
+    .bgStatus .currentDirection {
+        font-size: 60px;
+        line-height: 60px;
+    }
+
+    .bgStatus .currentDetails {
+        font-size: 15px;
+    }
+
+    #silenceBtn * {
+      font-size: 20px;
+    }
+
+    .time {
+        padding-top: 0;
+        font-size: 20px !important;
+    }
+
+    .timebox {
+        width: auto;
+    }
+
+    #currentTime {
+        display: inline;
+        vertical-align: middle;
+    }
+
+    .timeOther {
+        display: inline;
+        margin-left: auto;
+        font-size: 15px;
+    }
+
+    #uploaderBattery label {
+        font-size: 15px !important;
+    }
+
+    .focus-range {
+        position: absolute;
+        top: 80px;
+        left: initial;
+        right: 10px;
+        margin: 0;
+        width: auto;
+    }
+
+    .focus-range li {
+        display: block;
+    }
+
+    #chartContainer {
+        top: 190px;
+    }
+    #chartContainer svg {
+        height: calc(100vh - (190px));
+    }
 }
 
 @media (max-height: 700px) {
-  #chartContainer {
-    font-size: 14px;
-  }
+    #chartContainer {
+        font-size: 14px;
+    }
 }
 
 @media (max-height: 600px) {
-  #chartContainer {
-    font-size: 12px;
-  }
+    #chartContainer {
+        font-size: 12px;
+    }
 }
 
 @media (max-height: 480px) {
-  #toolbar {
-    float: right;
-    height: auto;
-  }
+    #toolbar {
+        float: right;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Use of !important](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/969293254/issues/source?bid=2452374&fileBranchId=2861014#l537)